packages/gatsby-plugin-image/src/components/__tests__/hooks.ts

```diff
@@ -0,0 +1,109 @@
+import { Node } from "gatsby"
+import { getSrc, getSrcSet, getImage, IGatsbyImageData } from "../../"
+
+const imageData: IGatsbyImageData = {
+  images: {
+    fallback: {
+      src: `imagesrc.jpg`,
+      srcSet: `imagesrcset.jpg 1x`,
+    },
+  },
+  layout: `constrained`,
+  width: 1,
+  height: 2,
+}
+
+const node: Node = {
+  id: ``,
+  parent: ``,
+  children: [],
+  internal: {
+    type: ``,
+    contentDigest: ``,
+    owner: ``,
+  },
+}
+
+const dataParent = {
+  ...node,
+  gatsbyImageData: imageData,
+}
+
+const fileNode = {
+  ...node,
+  childImageSharp: dataParent,
+}
+
+describe(`The image helper functions`, () => {
+  it(`getImage returns the same data if passed gatsbyImageData`, () => {
```

Comment:
Non-blocking, but I might make a nested describe for `getImage` since that's technically the object of the next few tests.